packages/compass-components/src/components/index-keys-badge.tsx

```diff
@@ -11,6 +11,8 @@ const keyListStyles = css({
 });
 
 const badgeStyles = css({
+  // Override LeafyGreen's uppercase styles as we want to keep the case sensitivity of the key.
+  textTransform: 'none',
```

Comment:
Is there a ticket for a proper prop that we can pass to the LG component? I think it might be good to link that here (and create one if it doesn’t currently exist)

Reply:
Asked in designs https://mongodb.slack.com/archives/GDTJXPHD0/p1688140011504569?thread_ts=1688049533.185989&cid=GDTJXPHD0
Good q, I have a feeling we probably shouldn't be using the LeafyGreen `Badge` here, and rather do something similar in our own code. Going to merge this so we get it into the next beta/ga, will follow up and add a todo with a ticket when there's a response. (Or will implement our own kind of Badge here).